common/config/config.json.template

```diff
@@ -74,7 +74,7 @@
       "wowzaUrl": "http://kalsegsec-a.akamaihd.net/dc-0/m/pa-live-publish2/kLive/smil:1_oorxcge2_all.smil/playlist.m3u8"
   },
   "mediaServer" : {
-      "hostname" : "@HOSTNAME@",
+      "hostname" : "HOSTNAME",
       "applicationName" : "kLive",
```

Comment:
this will break everything,....

there is a speicifc code  that changes @hostname@ to the actual host name, please revert